sources/engine/Stride.Audio/AudioEngine.cs

```diff
@@ -24,10 +24,7 @@ public class AudioEngine : ComponentBase
 
         static AudioEngine()
         {
-            if (!AudioLayer.Init())
-            {
-                throw new Exception("Failed to initialize the audio native layer.");
-            }
+            AudioLayer.Init();
```

Comment:
Maybe it's here that we could have a lazy loading. Just a thought, not to be done in that PR but in another one later for #2628.

Reply:
When I think about it it would be nice to lazy load other optional systems like VRDeviceSystem, DebugTextSystem, GameprofilingSystem etc. I'll try to concider such PR 🤔